src/renderer.cpp

```diff
@@ -311,13 +311,21 @@ Renderer::Renderer(const char *fontPath)
         }
 
         { // Load nanovg font
-            int font = nvgCreateFont(mNanoVg, "sans", fontPath);
+            int font = nvgCreateFont(mNanoVg, "robotoR", regularFontPath);
             if (font == -1)
             {
                 LOG_ERROR("Could not add font regular");
                 // TODO: Don't throw
                 throw Oddlib::Exception("Failed to load fonts");
             }
+            // Load nanovg font italics
+            font = nvgCreateFont(mNanoVg, "robotoI", italicFontPath);
+            if (font == -1)
+            {
+                LOG_ERROR("Could not add font italic");
+                // TODO: Don't throw
+                throw Oddlib::Exception("Failed to load fonts");
```

Comment:
this exception message might be a bit generic now there is 2 fonts? "Failed to load robotoR font" and "Failed to load robotoI font" perhaps?